main/tests/cypress/cypress.json

```diff
@@ -2,8 +2,8 @@
   "integrationFolder": "./cypress/integration",
   "nodeVersion": "system",
   "retries": {
-    "runMode": 2,
-    "openMode": 1
+    "runMode": 3,
+    "openMode": 0
```

Comment:
@fgiroud Why is there need to reduce retry in openMode? It helps in debugging the test through cypress binary.

Reply:
@kushthedude I found it not practical to have local retries, if some retries are necessary locally, it means there is probably an issue.
When a test fail, the dev needs to investigate quickly and retry manually after fix, local retries make the process cumbersome
Retries should be kept for CI/CD